rules/cross-platform/defense_evasion_attempt_to_disable_syslog_service.toml

```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/04/27"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+Adversaries may attempt to disable the Syslog service in an attempt to an attempt to disrupt event logging and evade
+detection by security controls.
+"""
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Attempt to Disable Syslog Service"
+note = "For macOS systems, Auditbeat must be run with an account that has access to the binaries in the /usr/sbin, /sbin, /usr/bin, and /bin directories. For Linux systems, to avoid conflicts with auditd, you will need to stop and disable auditd with 'sudo service stop auditd && sudo systemctl disable auditd'."
+risk_score = 47
+rule_id = "2f8a1226-5720-437d-9c20-e0029deb6194"
+severity = "medium"
+tags = ["Elastic", "Linux", "macOS"]
+type = "query"
+
+query = '''
+event.category:process and event.type:start and
+(process.args:(launchctl and unload and /System/Library/LaunchDaemons/com.apple.syslogd.plist))
+or (((process.name:service and process.args:stop) or (process.name:chkconfig and process.args:off)
+or (process.name:systemctl and process.args:(disable or stop or kill))) and process.args:(syslog or
+rsyslog or "syslog-ng"))
```

Comment:
Original query for reference

```sql
event.category:process and event.type:(start or process_started) and
  ((process.name:service and process.args:stop) or
     (process.name:chkconfig and process.args:off) or
     (process.name:systemctl and process.args:(disable or stop or kill)))
  and process.args:(syslog or rsyslog or "syslog-ng")
```

I think this is really what you were after - I also added support for `launchctl` service commands:
```suggestion
event.category:process and event.type:(start or process_started) and
  process.name:launchctl and process.args:(unload and /System/Library/LaunchDaemons/com.apple.syslogd.plist) or 
  
  ((process.name:service and process.args:stop) or
   (process.name:chkconfig and process.args:off) or
   (process.name:(systemctl or launchctl) and process.args:(disable or stop or kill))) and 
   process.args:(syslog or syslogd or rsyslog or "syslog-ng")
```

Reply:
When this is run on macOS, the `process.name` is `sudo`, not `launchctl`. 

With both Auditbeat and the Elastic Endpoint, `event.type:start` works because I excluded `event.action`, which isn't needed (everything is `start`) for either Auditbeat or the Endpoint.

Auditbeat
![image](https://user-images.githubusercontent.com/7442091/92034564-061e8580-ed33-11ea-90d1-03d01f5664e0.png)

Elastic Endpoint
![image](https://user-images.githubusercontent.com/7442091/92034594-17679200-ed33-11ea-9a07-a50154da4661.png)

Reply:
There should always be two process events for a sudo command:

one with `sudo` as the process.name that runs as the user.name of the user that invoked the command. And one with the target `process.name` with the user.name of `root`